src/django_components/component.py

```diff
@@ -99,6 +101,10 @@
 CssDataType = TypeVar("CssDataType", bound=Mapping[str, Any])
 
 
+# Keep track of all the Component classes created, so we can clean up after tests
+ALL_COMPONENTS: List[ReferenceType[Type["Component"]]] = []
```

Comment:
When keeping track of globally which component classes were defined, these are not stored as weak references, to allow the classes to be garbage collected if user decides to delete them.